index.js

```diff
@@ -166,7 +166,7 @@ module.exports = async function (url, options) {
                     return resolve(response);
                 }
 
-                console.error(`Retrying in ${retryOptions.retryInitialDelay} milliseconds, attempt ${attempt - 1} failed (status ${response.status}): ${response.statusText}`);
+                console.error(`Retrying ${url} in ${retryOptions.retryInitialDelay} milliseconds, attempt ${attempt - 1} failed (status ${response.status}): ${response.statusText}`);
```

Comment:
We actually avoid logging the raw url since it could contain sensitive information. Please redact the sensitive information from the url like this: https://github.com/adobe/asset-compute-commons#examples-2

Reply:
@jdelbick is there any other way, where we can pass a flag and enable logging or debugging urls?

And let user decide if they want to log or nt

Reply:
Yes, I would suggest using an environment variable (defaulting to not logging the url).